File: 9-palindrome-number/palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {   
    if (x < 0){
        return false
    }
    
    let n = x, rev = 0
    while(n){
        let tmp = n % 10
        rev = rev * 10 + tmp
        n = parseInt(n / 10) 
    }

    return x == rev
};

File: 9-palindrome-number/palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x < 0:
            return False
        
        n = x
        rev = 0
        while n > 0:
            tmp = n % 10
            rev = rev*10 + tmp
            n //= 10
        
        return rev == x